pkg/cmd/cmd_run.go

```diff
@@ -79,6 +80,13 @@ func NewRunCmd() *cli.Command {
 
 func KpmRun(c *cli.Context) error {
 	kclOpts := CompileOptionFromCli(c)
+	// set the pwd as the work dir.
+	pwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	kclOpts.Merge(kcl.WithWorkDir(pwd))
```

Comment:
1. The code here is somewhat tricky, and it may be part of the CompileOptionFromCli function
2. Workdir should be set to an absolute path